R/check.R

```diff
@@ -10,115 +10,48 @@
 #'
 #' @family check
 check_quantiles <- function(posterior, req_probs = c(0.5, 0.95, 0.2, 0.8)) {
-  if (any(req_probs <= 0) || any(req_probs >= 1)) {
-    stop("Please provide probabilities as numbers between 0 and 1.")
-  }
-  cols <- colnames(posterior)
-  if (sum(cols %in% paste0("q", req_probs * 100)) != length(req_probs)) {
-    stop(
-      "Following quantiles must be present (set with probs): ",
-      toString(req_probs)
-    )
-  }
-  return(invisible(NULL))
+  stopifnot(
```

Comment:
Does this feel like we are losing clarity in the error messages without a massive upside? What do you think?

Reply:
This is the same error message + condition? Or do you mean relative to the quantile bit?

I have contemplating some error message template options for `coerce_dt` and then having it use those.

Reply:
> ? Or do you mean relative to the quantile bit?

Yes

Reply:
righto, addressing this by allowing coerce_dt to have more specific error messages (passed by argument)

Reply:
Lovely.